packages/jest-circus/src/legacy_code_todo_rewrite/jest-adapter-init.js

```diff
@@ -43,6 +43,32 @@ const initialize = ({
   global.fit = global.it.only;
   global.fdescribe = global.describe.only;
 
+  global.test.concurrent = (
```

Comment:
there's too many changes that need to be made before we can do that.
it's easier to keep it as it is right now and we'll focus on redesigning it later